shesmu-server/src/test/resources/run/group-onlyif.shesmu

```diff
@@ -1,6 +0,0 @@
-Version 1;
-Input test;
-
-Olive
- Group By workflow Into a = OnlyIf `project`
```

Comment:
We use the pattern ``Group By workflow Into a = OnlyIf For i In `dictionary["key"]? As string`: Univalued i``

I'm looking at updating `OnlyIf` to `OnlyIf All` but it appears that `For` isn't supported with `OnlyIf All/Any` or at least not in the simulator?

Reply:
I'm assuming `dictionary[key]` is a single value, not a list, then the following should be equivalent:

```
Group By workflow Into a = `Univalued dictionary["key"]? As string`
```

Reply:
Added a related test case that uses some JSON conversion.